ext/nnstreamer/tensor_decoder/tensordec-directvideo.c

```diff
@@ -40,11 +41,59 @@ void fini_dv (void) __attribute__ ((destructor));
     GST_VIDEO_CAPS_MAKE ("{ RGB, BGRx, GRAY8 }") \
     ", views = (int) 1, interlace-mode = (string) progressive"
 
+/**
+ * @brief There can be different schemes for bounding boxes.
+ */
+typedef enum
+{
+  DIRECT_VIDEO_FORMAT_RGBx = 0,
+  DIRECT_VIDEO_FORMAT_BGRx = 1,
+  DIRECT_VIDEO_FORMAT_xRGB = 2,
+  DIRECT_VIDEO_FORMAT_xBGR = 3,
+  DIRECT_VIDEO_FORMAT_RGBA = 4,
+  DIRECT_VIDEO_FORMAT_BGRA = 5,
+  DIRECT_VIDEO_FORMAT_ARGB = 6,
+  DIRECT_VIDEO_FORMAT_ABGR = 7,
+  DIRECT_VIDEO_FORMAT_I420 = 8
```

Comment:
I420 is improper

Reply:
it's moved to channel 1 and other formats have been updated